src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -940,6 +940,11 @@ namespace Microsoft.Management.Deployment
         [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 12)]
         {
             /// Updates the package catalog.
+            /// The progress value, represented as a double, indicates the percentage of operation completion.
+            /// For "Microsoft.PreIndexed.Package" Package Catalogs, it reflects the progress of downloading and
+            /// installing the index. When there is no meaningful progress to report for other Package Catalogs,
+            /// the progress will be set to 0% at the start and will be updated to 100% upon the operation's completion.
+            /// The progress range is from 0 to 100.

```

Comment:
```suggestion
            /// The progress value, represented as a double, indicates the percentage of operation completion.
            /// The progress range is from 0 to 100.
```
I meant only for you to document the visible portion (range and meaning).  We should not document specifics on any of the internal implementation details.

This applies to the other progress comments as well.